client/extensions/woocommerce/state/sites/products/actions.js

```diff
@@ -159,25 +162,34 @@ export const fetchProducts = ( siteId, page ) => ( dispatch, getState ) => {
 		siteId = getSelectedSiteId( state );
 	}
 
-	if ( areProductsLoading( state, page, siteId ) ) {
+	const params = {
+		page,
+		per_page: 10,
+	};
+
+	if ( areProductsLoading( state, params, siteId ) ) {
 		return;
 	}
 
 	const fetchAction = {
 		type: WOOCOMMERCE_PRODUCTS_REQUEST,
 		siteId,
-		page,
+		params,
 	};
 	dispatch( fetchAction );
 
-	return request( siteId ).getWithHeaders( `products?page=${ page }&per_page=10` ).then( ( response ) => {
+	const paramString = Object.keys( params ).map(
```

Comment:
You can `import qs from 'querystring';` and do `qs.stringify( params )`.

Reply:
Thanks, I'll look at using that instead then.

Reply:
Fixed in cddd8c6